ovirt-release-host-node.spec.in

```diff
@@ -180,8 +180,7 @@ systemctl restart cockpit.service >/dev/null 2>&1
 
 PYTHON=$(command -v python3 || command -v python)
 
-pushd %{_sysconfdir}/yum.repos.d
-for REPO in $(find -maxdepth 1 -type f -name 'CentOS-oVirt-*.repo' -printf '%f\n');
```

Comment:
@lveyde not sure about this usage of find compared to master code, can you please review?

Reply:
It looks OK to me, but we'll of course will need to test it.

Reply:
I'm actually added the find code, that was removed by that patch, but it misses the search ovirt-*.repo files

Reply:
Sent PR#247 to sync the code in master branch with the code in ovirt-4.5 branch.